tests/integration_test/integration_test.sh

```diff
@@ -376,28 +385,38 @@ function stop_lira_on_error {
 trap "stop_lira_on_error" ERR
 
 # 8. Send in notifications
-printf "\n\nGetting notification token\n"
 
-notification_token=$(docker run -i --rm \
-      -e VAULT_TOKEN=$(cat $vault_token_path) \
-      broadinstitute/dsde-toolbox \
-      vault read -field=notification_token secret/dsde/mint/$env/listener/listener_secret)
+if [ "$use_hmac" == "true" ]; then
+  printf "\n\nGetting hmac key\n"
+  HMAC_KEY=$(docker run -i --rm \
+        -e VAULT_TOKEN=$(cat $vault_token_path) \
+        broadinstitute/dsde-toolbox \
+        vault read -field=current_key secret/dsde/mint/$env/listener/hmac_keys)
+  AUTH_PARAMS="--hmac_key $HMAC_KEY --hmac_key_id current_key"
```

Comment:
I haven't finish reading through the [protocol](https://tools.ietf.org/html/draft-cavage-http-signatures-10#section-1), so I'm not sure the relationships between preshared key and preshared key id here. Is it expected that we use the literals `current_key` as the hmac preshared key id here? (I assume the answer is yes since in the previous line the vault command's field is current_key)

Reply:
The key id given to send_notification.py is arbitrary because Lira only has one key in its config at any given time, so always uses that one and ignores the id. Storing it in vault as "current_key", so using that id here.